src/Foundation.hs

```diff
@@ -172,24 +173,7 @@ instance Yesod App where
     -- Default to Authorized for now.
     isAuthorized _ _ = return Authorized
 
-    -- This function creates static content files in the static folder
-    -- and names them based on a hash of their content. This allows
-    -- expiration dates to be set far in the future without worry of
-    -- users receiving stale content.
-    addStaticContent ext mime content = do
-        master <- getYesod
-        let staticDir = appStaticDir $ appSettings master
-        addStaticContentExternal
-            minifym
-            genFileName
-            staticDir
-            (StaticR . flip StaticRoute [])
-            ext
-            mime
-            content
-      where
-        -- Generate a unique filename based on the content itself
-        genFileName lbs = "autogen-" ++ base64md5 lbs
+    addStaticContent = embedStaticContent appStatic StaticR minifym
```

Comment:
There are two problems here.

1. Static files are currently not installed. To make sure they are installed by `cabal install` we need to add all static files to the `data-files` section in `pursuit.cabal`
2. The second issue is `addStaticContentExternal` today is going to minify the static assets and drop `tmp/autogen-*` files into the static files directory. This breaks on systems where the installation directory is read-only, like the `/nix/store` on NixOS.

Embedding the static assets into the compiled binary fixes both issues.